packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -0,0 +1,735 @@
+            );
+            await longNextTick();
+
+            expect(bold.parentNode.childNodes.length).to.equal(5);
+            expect(bold.previousElementSibling.classList.contains('self-closing')).to.equal(
+                true,
+                'previous is the range start',
+            );
+            expect(bold.nextElementSibling.classList.contains('self-closing')).to.equal(
+                true,
+                'next is the range end',
+            );
+
+            const selection = document.getSelection();
+            expect(selection.rangeCount).to.equal(1, 'range exist');
+            const range = selection.getRangeAt(0);
+
+            const b = editor.el.querySelector('test-container b');
+            expect(range.startContainer).to.equal(b.previousSibling);
+            expect(range.startOffset).to.equal(1);
+            expect(range.endContainer).to.equal(b.firstChild);
+            expect(range.endOffset).to.equal(1);
+        });
+        it('should insert a char in editable (to add commands)', async () => {
+            const container = editor.el.querySelector('test-container');
+            container.dispatchEvent(new KeyboardEvent('keydown', { key: 'z' }));
+            container.querySelector('b').firstChild.textContent = 'z';
+            container.dispatchEvent(new KeyboardEvent('input', { bubbles: true }));
+            await nextTick();
+            await nextTick();
+            expect(container.querySelector('b').parentElement.innerHTML).to.equal('a<b>z</b>c');
+
+            await longNextTick();
+            const bold = devtools.querySelector('devtools-node .selectable-line.bold');
+            expect(bold.parentElement.parentElement.textContent).to.equal('az[]c');
+        });
+    });
+    describe('PathComponent', () => {
+        it('should select a parent node with the bottom path bar', async () => {
+            const path = devtools.querySelector('devtools-path devtools-pathnode:nth-child(2)');
+            path.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+            await longNextTick();
+            const p = devtools.querySelector(
+                'devtools-node .children > devtools-node.element:nth-child(2)',
+            );
+            expect(p.firstElementChild.classList.contains('selected')).to.equal(true);
+        });
+    });
+    describe('InfoComponent', () => {
+        it('should select a child node with the info panel', async () => {
+            let about = devtools.querySelector('devtools-info .about');
+            let aResult =
+                '<div class="about">' +
+                '<span class="type">VNode</span> PARAGRAPH<span class="id">9</span>' +
+                '</div>';
+            expect(about.outerHTML).to.equal(aResult);
+
+            const li = [].slice
+                .call(devtools.querySelectorAll('devtools-info .properties li'))
+                .pop();
```

Comment:
Why not use Array.prototype.slice if you really want to do that ? Otherwise, you could also do `Array.from(...).slice().pop()`